h/static/styles/partials-v2/_search.scss

```diff
@@ -106,7 +106,7 @@
 // Card displaying stats about a group of annotations in search results
 .search-bucket-stats {
   @include font-normal;
-  width: 210px;
+  width: 260px;
```

Comment:
Ah, I hadn't thought to test this with different screen sizes! Now I wonder if this bit of layout needs a larger refactoring, as you say there seems to be multiple problems with smaller screen sizes.

Here it is with `max-width:260px` at the maximum width:

![screenshot from 2016-09-26 11-15-25](https://cloud.githubusercontent.com/assets/22498/18830995/3c154414-83db-11e6-8055-57f3c6b24acd.png)

But it does start to look quite bad as the window gets smaller (stats retaining its width even though I used `max-width` not `width`), and the actual annotation card shrinking), this is as bad as it gets:

![screenshot from 2016-09-26 11-15-53](https://cloud.githubusercontent.com/assets/22498/18830928/ef73f7a4-83da-11e6-80d0-7579d4f93a20.png)

If the window gets any smaller than that then it hits a breakpoint where the stats move below the annotations. Notice the apparently pointless big empty gap on the left hand side, annotation card pointlessly squished (when there is a big empty gap to the left of it), and misalignment of the annotation card and the title:

![screenshot from 2016-09-26 11-16-15](https://cloud.githubusercontent.com/assets/22498/18830974/29092066-83db-11e6-8222-59d3dd11f0e8.png)

This all gets worse as the window gets smaller:

![screenshot from 2016-09-26 11-21-41](https://cloud.githubusercontent.com/assets/22498/18831035/7b6a37f0-83db-11e6-9395-a442ea6c6e16.png)